cmd/es-rollover/app/init/action.go

```diff
@@ -84,7 +85,8 @@ func (c Action) Do() error {
 func createIndexIfNotExist(c client.IndexAPI, index string) error {
 	err := c.CreateIndex(index)
 	if err != nil {
-		if esErr, ok := err.(client.ResponseError); ok {
+		var esErr client.ResponseError
```

Comment:
Does this not be a pointer `*client.ResponseError`? E.g. this doesn't work if you remove the pointer https://go.dev/play/p/-mL6xwpYpIa

Reply:
nvm, this is correct